libs/elasticsearch-core/src/main/java/org/elasticsearch/common/Booleans.java

```diff
@@ -107,7 +107,7 @@ public static Boolean parseBoolean(String value, Boolean defaultValue) {
     }
 
     /**
-     * Returns <code>false</code> if text is in <tt>false</tt>, <tt>0</tt>, <tt>off</tt>, <tt>no</tt>; else, true
+     * Returns {@code false} if text is in "false", "0", "off", "no"; else, {@code true}.
```

Comment:
`<code>` is still valid but `{@code}` is the "standard" way to do it in Javadoc and i was converting the `<tt>`s on the same line. I certainly wouldn't go out of my way to replace `<code>` with `{@code}` but this one was right there....

Reply:
Personally I think that “null”, “true”, and “false” do not need to be wrapped in`{@code }` formatting. They occur so frequently it becomes a burden for the author with little if any benefit for the reader. This is not a comment to invoke change, only a comment.

Reply:
Fair enough. I like how it offsets the rendered text but when reading it un-rendered it doesn't add much.